frontend/amundsen_application/api/utils/metadata_utils.py

```diff
@@ -102,6 +102,12 @@ def is_table_editable(schema_name: str, table_name: str, cfg: Any = None) -> boo
     return True
 
 
+def _set_type_metadata_is_editable(type_metadata: TypeMetadata, is_editable: bool) -> None:
+    type_metadata['is_editable'] = is_editable
+    for tm in type_metadata['children']:
+        _set_type_metadata_is_editable(tm, is_editable)
```

Comment:
so right now editability is only controlled down to the parent level? Lets say we want to set a nested column to be have an editable description we can't then set one of it's sub column's to not have an editable description right?

Reply:
That's true as of now, I followed how the columns are handled as well, since it appears that as of now it only has table level configurations to determine whether columns are editable or not. I think this could be adjusted in the future pretty easily if that were to change